pkg/hubble/container/ring_reader_test.go

```diff
@@ -94,7 +95,7 @@ func TestRingReader_Previous(t *testing.T) {
 			var got []*v1.Event
 			for i := 0; i < tt.count; i++ {
 				event, err := reader.Previous()
-				if err != tt.wantErr {
+				if !errors.Is(err, tt.wantErr) {
```

Comment:
I don't think we want these changes to be applied in this test file. This tests that the error return is _exactly_ `ErrInvalidRead` or `io.EOF`, not just somewhere in the errors chain. Semantically, it's not the same thing.